tests/test_arithmetics.rs

```diff
@@ -233,6 +233,13 @@ fn binary_arithmetic() {
         py_expect_exception!(py, c, "1 ** c", PyTypeError);
 
         py_run!(py, c, "assert pow(c, 1, 100) == 'BA ** 1 (mod: Some(100))'");
+
+        let c: Bound<'_, PyAny> = c.extract().unwrap();
+        assert_eq!(c.add(&c).unwrap().extract::<&str>().unwrap(), "BA + BA");
+        assert_eq!(
+            c.pow(&c, py.None()).unwrap().extract::<&str>().unwrap(),
+            "BA ** BA (mod: None)"
+        );
```

Comment:
I wonder if we can write a helper to make this sort of check easier, e.g.

```suggestion
        assert_py_eq!(c.add(&c).unwrap(), "BA + BA");
```

... which I guess calls `.eq("BA + BA").unwrap()`.

It might also be nice to list out all these methods again explicitly, even if it's a bit tedious.

Reply:
Good idea.